src/applications/personalization/profile-2/components/Profile2Router.jsx

```diff
@@ -125,64 +126,69 @@ class Profile2Router extends Component {
 
     return (
       <BrowserRouter>
-        <Profile2Wrapper
-          routes={routes}
-          isLOA3={this.props.isLOA3}
-          isInMVI={this.props.isInMVI}
-        >
-          <Switch>
-            {/* Redirect users to Account Security to upgrade their account if they need to */}
-            {routes.map(route => {
-              if (
-                (route.requiresLOA3 && !this.props.isLOA3) ||
-                (route.requiresMVI && !this.props.isInMVI)
-              ) {
+        <LastLocationProvider>
+          <Profile2Wrapper
```

Comment:
the only change here is wrapping the `Profile2Wrapper` in a `LastLocationProvider`